### DISCUSSION TITLE
Self Host Hubs Cloud

### DISCUSSION THREAD

**Discussion by WolfTown0907** [mozilla/hubs#3554] [Hubs Cloud Questions] [answered]
Hello.
I would like to know if I can self-host Hubs Cloud without using a droplet, but hosted natively on my own server. If this is possible, can someone explain how to do so?

**Comment by robinkwilson** [comment, marked as answer]
Great question! Though a duplicate question. Search for `self hosting` on discussions to pull these up. Hope these help!

Relevant discussion threads:
https://github.com/mozilla/hubs/discussions/3099
https://github.com/mozilla/hubs/discussions/2699
https://github.com/mozilla/hubs/discussions/3049

**Comment by albirrkarim** [comment]
I think my [article about hosting hubs on VPS](https://github.com/albirrkarim/mozilla-hubs-installation-detailed/blob/main/VPS_FOR_HUBS.md) can help you
